### ISSUE TITLE
Cannot set properties of null (setting 'scrollTop')

### ISSUE THREAD

**Issue by lrnselfreliance** [mkdocs/mkdocs#3773]
I encounter an error when attempting to navigate to the second level of the navigation menu.  Here is a video:

https://github.com/user-attachments/assets/88a9ebd8-119c-45d7-a13e-d83133413774

Navigation to both `docs/one/a/index.md` and `docs/one/b/index.md` works fine using links in the root index.  This error only occurs with both `a` and `b` directory.  The error goes away if you delete the `b` directory.

I have a much larger mkdocs project which displays the same error.  The larger project has many more files, and several level two navigation menus.

This error still occurs with latest master 62a4e3c.

### requirements.txt
```
mkdocs==1.6.0
````

### mkdocs.yml
```yaml
site_name: test
theme:
  name: mkdocs
  color_mode: auto
  user_color_mode_toggle: true
```

### docs/index.md
```md
# Index

[A Index](one/a/index.md)

[B Index](one/b/index.md)
```

### docs/one/index.md
```md
# One Index
```

### docs/one/a/index.md
```md
# A Index
```

### docs/one/b/index.md
```md
# B Index
```

## Error

Browser error:

```
Uncaught TypeError: popup.querySelector(...) is null
    hideInnerDropdown http://127.0.0.1:8000/js/base.js:121
    showInnerDropdown http://127.0.0.1:8000/js/base.js:96
    showInnerDropdown http://127.0.0.1:8000/js/base.js:94
    <anonymous> http://127.0.0.1:8000/js/base.js:130
```

**Comment by pawamoy** [comment]
Kinda randomly tagging @squidfunk who helpfully replaced jQuery with vanilla JavaScript previously (sorry)

**Comment by squidfunk** [comment]
I think this was already fixed in https://github.com/mkdocs/mkdocs/pull/3694, but not released yet. Could you please check if this errors occurs in the latest `master` of MkDocs? It would be great to have those bug fixes released, but I can't do it, since I'm not the maintainer here.

**Comment by squidfunk** [comment]
Fixed in #3774. This should be one of the last instances where `querySelector` throws. The reason was that jQuery, before, just seemed to have converted this to a no-op, so chained calls just did nothing.

IMHO, the script needs a proper refactoring. I helped to remove jQuery in #3649, so some issues could be closed, but maybe Tom can tackle this as part of the theme rebrush he's currently working at (see #3681 for instance) ☺️
